experimental/ComplexReflectionGroups/src/symplectic_reflection_group.jl

```diff
@@ -0,0 +1,38 @@
+# This file implements explicit models of symplectic reflection groups.
+#
+# Ulrich Thiel, 2024
+
+export symplectic_reflection_group
+export is_symplectic_reflection_group
+
+function symplectic_reflection_group(W::MatrixGroup)
+
+    if !is_complex_reflection_group(W)
+        throw(ArgumentError("Group must be a complex reflection group"))
```

Comment:
For what this function is doing, not really... (somebody should invent a good name for this 'doubling operation').

Reply:
It will make sense in the greater scheme: symplectic_reflection_group(...) will create a symplectic reflection group for a labeling system you will come up with. And then this function will be the convenience constructor for the one cooked up from a complex reflection group...Something like that...

Reply:
Btw I always called this 'doubling operation' 'symplectic doubling' but it seems no one (in particular you) likes it...